circe-akka-serializer/src/main/scala/org/virtuslab/ash/Register.scala

```diff
@@ -0,0 +1,12 @@
+package org.virtuslab.ash
```

Comment:
Shouldn't the package for all things Circe-serializer-related be sth more specific than just `org.virtuslab.ash`? Esp. `org.virtuslab.ash.Register` feels very non-specific to Circe